src/cache_manager.cc

```diff
@@ -248,6 +248,8 @@ CacheManager::ParseHeaders(const HttpRequest * request, Mgr::ActionParams &param
 
```

Comment:
> PR description: Reporters ... can now determine their content-type support dynamically and based on Accept header
inputs to something different from text/plain.

Something went wrong in that PR description edit -- the "to something different from text/plain" phrase is not attached to anything I can detect. I suggest to simplify: "Actions can now vary their responses based on the Accept request header."